### ISSUE TITLE
[LCN] Adding action for "LCN_MRS" or "PCK Commands"

### ISSUE THREAD

**Issue by cx555** [openhab/openhab-addons#11250]
Hi,

Is it possible to implement the control for LCN-MRS or PCK commands?
Hitkey action is not a nice solution for volume adjustment or input switching for 8 inputs and 13 outputs.

Thanks

**Comment by fwolter** [comment]
Yes, it is possible. But as it's discontinued and I don't have one either, I'm not highly motivated to implement the commands. I'm wondering if there are any disadvantages using the Hitkey Action?

**Comment by cx555** [comment]
Hello,
that's a shame.  I know it was discontinued but I bought the MRS back then.  It is possible to use HITKEY e.g.  for + -5 louder / quieter, but I have no real feedback on the volume.  And to change the Chennels you need a lot of key tables in the LCN PRO.  Even if you don't have the MRS, PCK commands would be very helpful to control the MRS and I think for other users too.
